armi/nucDirectory/nuclideBases.py

```diff
@@ -16,6 +16,12 @@
 The nuclideBases module classes for providing *base* nuclide information, such as
 Z, A, state and energy release.
 
+By default the data used to create the nuclides comes from a subset of the RIPL-3
+database, which ships with ARMI. The entire RIPL-3 database can be downloaded from
+`<https://www-nds.iaea.org/RIPL-3/levels/>`_. If the ``ARMI_RIPL_PATH`` environment
+is set to a location containing the full RIPL-3 database, all nuclides will be read
+from there.
+
```

Comment:
Now that I think about it, some justification as to why we are operating by default with a pruned set of nuclides, and/or why one would want the full data set could be useful. Something to the tune of "For most reactor calculations, it is typical to reduce the dimensionality of depletion by ignoring or lumping together nuclides that aren't important. For other tasks, like detailed decay or activation analysis, a full set of nuclides may be more important."

Reply:
Added this to the `user_install.rst` file and added a link to that documentation here.